python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1696,6 +1696,53 @@ def test_parquet_writer_chunked_partitioned(tmpdir_factory, return_meta):
     assert_eq(got_pd, got_cudf)
 
 
+@pytest.mark.parametrize("max_file_size", [100, "10KB"])
+def test_parquet_writer_chunked_max_file_size(tmpdir_factory, max_file_size):
```

Comment:
my only question about this is that it seems like this checks that we roundtrip the data in the case where `max_file_size` is passed, but doesn't as far as I can see do any inspection of the file sizes on disk. is that something that is worth checking or could be done simply?

Reply:
Yup, added validation. I bumped the file sizes a bit since the current file sizes are too small for parquet file creation.